### PR TITLE
WSTEAM1-298: Restrict recommendations requests for sfv articles.

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves JIRA [WSTEAM1-298](https://jira.dev.bbc.co.uk/browse/WSTEAM1-298)

Overall changes
======
Prevents WSOJ from rendering on Article SFV pages.

Regular Article Page:
Screenshot 2023-03-08 at 15 36 02

Article SFV Page: 
Screenshot 2023-03-08 at 15 37 26

Code changes
======

- _getInitialData has been refactored to pull out the consumableAsSFV flag._
- _getOnwardsData has been modified to NOT request WSOJ data when consumableAsSFV is true._

Testing
======
1. _List the steps used to test this PR._

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/routes/article/getInitialData/index.test.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ const bffArticleJson = {
       content: {},
       metadata: {
         allowAdvertising: true,
+        consumableAsSFV: true,
```
Comment (karinathomasbbc) [inline src/app/routes/article/getInitialData/index.test.ts:19]: I just wanted to check - is this returned from the article fabl-module?

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/routes/article/getInitialData/index.test.ts (hunk 1 above)
Comment (shayneahchoon) [reply src/app/routes/article/getInitialData/index.test.ts:19]: It is indeed from the simorgh-bff module, sample response from FABL: 
Screenshot 2023-03-08 at 16 56 19

**Review by amoore108 — APPROVED** [review]
Nice!

**Review by Isabella-Mitchell — APPROVED** [review]
Looks good to me! I like the way you've moved logic out from the article/getInitialData into files in the article/utils folder.

**Pull request merged**